docs/BRP-LO GBA mapping - Personen gba.csv

```diff
@@ -0,0 +1,137 @@
+categorie;element;leveren door RvIG;Beperkt zoekresultaat;Persoon GBA variant;;;;;opmerking/issue;
+;;vaste waarde: GbaPersoon | GbaPersoonBeperkt;G;type;;;;;;
+1;01.10;ongewijzigd leveren;;aNummer;;;;;;
+1;01.20;ongewijzigd leveren;G;burgerservicenummer;;;;;;
+1;02;;;naam;;;;;;
+1;02.10;ongewijzigd leveren;G;;voornamen;;;;;
+1;02.20;code+omschrijving+soort;G;;adellijkeTitelPredicaat;;;;;
+1;02.30;ongewijzigd leveren;G;;voorvoegsel;;;;;
+1;02.40;ongewijzigd leveren;G;;geslachtsnaam;;;;;
+1;61.10;code+omschrijving;;;aanduidingNaamgebruik;;;;;
+1;03;;;geboorte;;;;;;
+1;03.10;ongewijzigd leveren (8 cijfers);G;;datum;;;;;
+1;03.20;code+omschrijving;;;plaats;;;;;
+1;03.30;code+omschrijving;;;land;;;;;
+1;04.10;code+omschrijving;G;geslacht;;;;;;
+1;83;;;persoonInOnderzoek;;;;;;
+1;83.10;ongewijzigd leveren (6 cijfers);;;aanduidingGegevensInOnderzoek;;;;;
+1;83.20;ongewijzigd leveren (8 cijfers);;;datumIngangOnderzoek;;;;;
+2, 3;;ook leveren volledig onbekende ouder;;ouders;;;;;;
+2, 3;;als categorie == 2: 1, als categorie == 3: 2;;;ouderAanduiding;;;;;
+2, 3;01.20;ongewijzigd leveren;;;burgerservicenummer;;;;;
+2, 3;02;;;;naam;;;;;
+2, 3;02.10;ongewijzigd leveren;;;;voornamen;;;;
+2, 3;02.20;code+omschrijving+soort;;;;adellijkeTitelPredicaat;;;;
+2, 3;02.30;ongewijzigd leveren;;;;voorvoegsel;;;;
+2, 3;02.40;ongewijzigd leveren;;;;geslachtsnaam;;;;
+2, 3;03;;;;geboorte;;;;;
+2, 3;03.10;ongewijzigd leveren (8 cijfers);;;;datum;;;;
+2, 3;03.20;code+omschrijving;;;;plaats;;;;
+2, 3;03.30;code+omschrijving;;;;land;;;;
+2, 3;04.10;code+omschrijving;;;geslacht;;;;;
+2, 3;62.10;ongewijzigd leveren (8 cijfers);;;datumIngangFamilierechtelijkeBetrekking;;;;;
+2, 3;83;;;;inOnderzoek;;;;;
+2, 3;83.10;ongewijzigd leveren (6 cijfers);;;;aanduidingGegevensInOnderzoek;;;;
+2, 3;83.20;ongewijzigd leveren (8 cijfers);;;;datumIngangOnderzoek;;;;
+4;;;;nationaliteiten;;;;;;
+4;05.10;code+omschrijving;;;nationaliteit;;;;;
+4;63.10;code+omschrijving;;;redenOpname;;;;;
+4;65.10;code+omschrijving;;;aanduidingBijzonderNederlanderschap;;;;;
+4;83;;;;inOnderzoek;;;;;
+4;83.10;ongewijzigd leveren (6 cijfers);;;;aanduidingGegevensInOnderzoek;;;;
+4;83.20;ongewijzigd leveren (8 cijfers);;;;datumIngangOnderzoek;;;;
+4;85.10;ongewijzigd leveren (8 cijfers);;;datumIngangGeldigheid;;;;;
+5;;actuele partner(s) OF de laatst ontbonden partner;;partners;;;;;ontbonden partners worden in GbaPersoon geleverd t.b.v. naamsamenstelling;
```

Comment:
deze opmerking klopt niet. Is nu de enige opmerking, dus hele kolom opmerking/issue kan dan vervallen